src/components/login/login.component.ts

```diff
@@ -20,7 +20,7 @@ export class LoginComponent {
               private router: Router) {
     // Tests if user already logged in
     this.backend.sendLoginData("", "").subscribe(res => {
-      if (res.status == 409) this.router.navigate(['/create-game']);
+      if (res.error == "already logged in") this.router.navigate(['/create-game']);
```

Comment:
Actually this is not possible, because res.error does not exist.
The message is saved in the private variable res._body.